x/lockup/keeper/gas_test.go

```diff
@@ -70,6 +70,6 @@ func (suite *KeeperTestSuite) TestRepeatedLockTokensDistinctDurationGas() {
 
 	avgGas, maxGas := suite.measureAvgAndMaxLockGas(totalNumLocks, defaultAddr, coinsFn, durFn)
 	fmt.Printf("test deets: total locks created %d\n", totalNumLocks)
-	suite.Assert().EqualValues(110725, int(avgGas), "average gas / lock")
-	suite.Assert().EqualValues(231313, int(maxGas), "max gas / lock")
+	suite.Assert().EqualValues(122316, int(avgGas), "average gas / lock")
+	suite.Assert().EqualValues(242903, int(maxGas), "max gas / lock")
```

Comment:
These constants are suspected to have gone up due to FundAccount instead of SetBalances.